_data/administrations/municipal.yml

```diff
@@ -4,13 +4,11 @@
   name:
     en: City of Montreal
     fr: Ville de Montréal
-
```

Comment:
Removed spaces because it was causing unnecessary diffs on commits when creating a new organization. Same goes for the comment.